src/UXClient/Models/ScatterPlotData.ts

```diff
@@ -15,6 +15,8 @@ class ScatterPlotData extends GroupedBarChartData {
     /******** SETS EXTENT OF EACH DATA MEASURE -- MEASURES UPDATED WHEN RENDER CALLED OUTSIDE OF TEMPORAL ********/
     public setExtents(measures: any, forceReset: boolean = false){
         if(!this.extentsSet || forceReset){
+            // Reset extents
+            this.extents = {};
```

Comment:
Added this logic to fix issue where tool-tip would show all measures added since construction.